mantidproject/fitbenchmarking
# Updated unit tests

#### Description of Work

Fixes #53 
Updated the unit tests such that they don't fail anymore.

#### Testing Instructions

1. In the `/fitbenchmarking/fitbenchmarking/tests/` folder, run the tests in the mantidpython console.
2. Make sure all the tests pass.
